website/members/models/member.py

```diff
@@ -99,39 +99,47 @@ def __str__(self):
         return f"{self.get_full_name()} ({self.username})"
 
     def refresh_from_db(self, **kwargs):
-        # Clear the cached latest_membership
-        if hasattr(self, "_latest_membership"):
-            del self._latest_membership
+        # Clear the cached current- and latest_membership
+        if hasattr(self, "_current_membership"):
+            del self._current_membership
+        if hasattr(self, "current_membership"):
+            del self.current_membership
         if hasattr(self, "latest_membership"):
             del self.latest_membership
 
         return super().refresh_from_db(**kwargs)
 
-    @property
+    @cached_property
```

Comment:
Whats the benefit of changing this to a cached property?

Reply:
It doesn't do the query multiple times if called multiple times and not prefetched

Reply:
Ah fair enough